src/python/python/ad/integrators/prbvolpath.py

```diff
@@ -278,10 +278,20 @@ def sample(self,
                                                                sampler.next_2d(act_medium_scatter),
                                                                act_medium_scatter)
                 act_medium_scatter &= phase_pdf > 0.0
+
+                # Re evaluate the phase function value in an attached manner
+                phase_eval, _ = phase.eval_pdf(phase_ctx, mei, wo, act_medium_scatter)
+                if not is_primal and dr.grad_enabled(phase_eval):
+                    Lo = phase_eval * dr.detach(dr.select(active, L / dr.maximum(1e-8, phase_eval), 0.0))
```

Comment:
Just to double-check, this would be "_more_" correct:
```suggestion
                    Lo = phase_eval * dr.detach(dr.select(act_medium_scatter, L / dr.maximum(1e-8, phase_eval), 0.0))
```
Right?
Both are functionally and numerically equivalent (because `phase_eval` will be a detached `0` if `act_medium_scatter` is `False` and `active` is True`), but as a reader it seems weird that the masking is different..

Reply:
Yes, you're right.